### DISCUSSION TITLE
Debug overlay is not readable when using scaling

### DISCUSSION THREAD

**Discussion by peacememories** [replit/kaboom#754] [Q&A]
Hi, and thanks for developing this great little engine.

We're developing a game in GBA resolution and are using 160*144 pixels, with a scaling of 4, to get to 640 px in width.

This works great for the game, but it seems to also affect the debug menu, which means that most of it is completely unreadable. I've dug through the code a bit, and it seems the debug view *does* use a form of `renderUnscaled` which resets the width and height to its full 640px temporarily, but it seems this does not mean a higher resolution for the debug interface.

Are we doing something wrong here, or is this intended behavior? I think it would make sense to render the debug UI at higher resolutions even for low-res games, to keep the text fully readable.

Thanks for your time :)

**Comment by lajbel** [comment]
That was fixed in #755
